packages/slice-machine/components/AppLayout/Navigation/Menu/Navigation/VideoItem.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import ReactTooltip from "react-tooltip";
+import Item from "@components/AppLayout/Navigation/Menu/Navigation/Item";
+import { MdPlayCircleFilled } from "react-icons/md";
+import { Close, Flex, Paragraph } from "theme-ui";
+import style from "./VideoItem.module.css";
+
+type VideoItemProps = {
+  viewedVideosToolTip: boolean;
+  onClose: () => void;
+};
+
+const VideoItem: React.FC<VideoItemProps> = ({
+  viewedVideosToolTip,
+  onClose,
+}) => {
+  const ref = React.createRef<HTMLParagraphElement>();
+  const id = "video-tool-tip";
+
+  const handleClose = () => {
+    onClose();
+  };
+
+  React.useEffect(() => {
+    if (!viewedVideosToolTip && ref.current) {
+      ReactTooltip.show(ref.current);
```

Comment:
does this needs to be here ?
Can't the store be initialised to `true` if it doesn't exist ?

Reply:
Which store you are talking about ?